javascript/javascript2/week1/index2js2week1.html

```diff
@@ -0,0 +1,79 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="main.css" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+    rel="stylesheet"
+  />
+</head>
+<body>
+  <main>
+    <header>
+      <div class="wrapper">
+        <h1>HyfBay</h1>
+        <h2>- Buy the okay'est products</h2>
+      </div>
+    </header>
+    <section class="table-view-control">
+      <div class="search">
+        <label>What are you looking for?</label>
+        <input
+          type="search"
+          id="searchName"
+          placeholder="Search for a product"
+        />
+        <label>Enter a price limit</label>
+        <input type="number" id="maxPrice" placeholder="Enter max price" />
+      </div>
+
+      <div class="sort">
+        <label>Sort:</label>
+        <select>
+          <option value="Default">Default</option>
+          <option value="LowToHigh">Low to high</option>
+          <option value="HighToLow">High to low</option>
+        </select>
+      </div>
+      <div class="filters">
+        <div class="country">
+          <label>Ships to</label>
+          <select>
+            <option value="denmark">Denmark</option>
```

Comment:
it could use another option "all countries" so the user can reset this filter - it is better user experience. Now we have to reload the page to see all countries again :)